Userland/Applications/Spreadsheet/Spreadsheet.cpp

```diff
@@ -307,16 +308,18 @@ void Sheet::copy_cells(Vector<Position> from, Vector<Position> to, Optional<Posi
     auto copy_to = [&](auto& source_position, Position target_position) {
         auto& target_cell = ensure(target_position);
         auto* source_cell = at(source_position);
+        cell_changes.append(make_ref_counted<CellChange>(target_cell, target_cell.data()));
```

Comment:
🤔 how would the redo work for this?

Reply:
I remember thinking it was clever at the time :D .. Since the target_cell is going to be changed regardless, and since the CellChange constructor takes the cell reference and the previous value, then can create the CellChange before actually changing the cell's value.